config/common/iatlas-vpc.yaml

```diff
@@ -0,0 +1,8 @@
+template_path: remote/vpc-mini.yaml
+stack_name: iatlas-vpc
+parameters:
+  VpcSubnetPrefix: "10.255.20"
+  VpcName: iatlas-vpc
```

Comment:
our convention is no `-`, change to `iatlasvpc` if you want to stick with our convention